### PR TITLE
[DUBBO-11047][issue]Remove direct dependency of protobuf-java

### PR DESCRIPTION
**Pull request by wlazjr**
## What is the purpose of the change

replace protobuf depency in triple rpc with customer encoding/decoding

## Brief changelog
add TripleCustomerProtocolWapper.java
 replace TripleRequestWrapper.TripleRequestWrapper with TripleCustomerProtocolWapper.TripleRequestWrapper
 replace TripleRequestWrapper.TripleResponseWrapper with TripleCustomerProtocolWapper.TripleResponseWrapper

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Review by EarthChen — CHANGES_REQUESTED** [review]
fix ut & it

**Comment by EarthChen** [comment]
If we remove the pb dependency, the built-in service also needs to be moved out to other modules, otherwise it will still fail to start due to the built-in service at startup.

**Comment by wlazjr** [comment]
> ### This branch has no conflicts with the base branch



> If we remove the pb dependency, the built-in service also needs to be moved out to other modules, otherwise it will still fail to start due to the built-in service at startup.

ok，got it

**Comment by guohao** [comment]
Changes:
1. Set `BuiltinService` to `disable` by default
2. Check the  `BuiltinService` if is `enable` in `TripleProtocol`
3. Add check condition at `Status` usages

File: dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/service/TriBuiltinService.java [hunk 1]
```diff
@@ -67,6 +74,11 @@ public void init() {
         }
     }
 
+    public boolean enable(){
+        return config.getBoolean(TRI_BUILTIN_SERVICE_INIT, false);
```
Comment (AlbumenJ) [inline dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/service/TriBuiltinService.java:78]: Can we check if users has add the `protobuf-java` dependency?

Comment (wlazjr) [reply dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/service/TriBuiltinService.java:81]: First of all, we do not know the real whole sets of versions that works fine, and in the future as protobuf-java update, we may need hava a glance at this; on the other hand, i think give the load permissions to user is a better choice.

**Review by guohao — APPROVED** [review]
LGTM

**Review by EarthChen — APPROVED** [review]
LGTM

**Pull request merged**